### ISSUE TITLE
PROJECT 1 - Hotdog / no hotdog

### ISSUE THREAD

**Issue by albertkjoller** [albertkjoller/DL-COMVIS#1]
MODEL:
- [ ] Get dataloader
- [ ] Define architecture
- [ ] Define optimizer
- [ ] Data augmentation

EXPERIMENTS:
- [ ] Experiment with different optimizers
- [ ] Experiment with data augmentation
- [ ] Experiment with batch normalization
- [ ] Get performance (accuracy) (maybe others, such as confusion matrix)
- [ ] Which images are classified wrongly?
- [ ] Compute SmoothGrad saliency maps (+ visualization of noise importance)

POSTER:
- [ ] Create poster
- [ ] Add results
- [ ] Write and discuss (see project description)
